app/controllers/hyrax/notifications_controller.rb

```diff
@@ -8,6 +8,8 @@ def index
       add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
       add_breadcrumb t(:'hyrax.admin.sidebar.notifications'), hyrax.notifications_path
       @messages = user_mailbox.inbox
+      # Update the notifications now that there are zero unread
+      StreamNotificationsJob.perform_later(current_user)
```

Comment:
This is the only place in the code where user notifications are marked as read. Calling the job here ensures the user's notifications clear out.